### PR TITLE
bank: cleaning expired sessions

### LINKED ISSUE #50: bank: cleaning expired sessions.
Expired sessions should be removed from the database.

### PR DESCRIPTION
**Pull request by martinmilev**
Expired sessions are removed from bd every 5 minutes.

> Fixes #50

### CODE REVIEW

File: src/test/java/com/clouway/bank/core/SessionCleanerTest.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+  @Before
+  public void setUp() throws Exception {
+    sessions = new PersistentSessionRepository(datastoreRule.getDatabase());
+  }
+
+  @Test
+  public void happyPath() {
+    Session session = sessions.startSession(LocalDateTime.of(2017, 1, 30, 9, 0),"::any username::");
+    SessionCleaner cleaner = new SessionCleaner(sessions, 1);
```
Comment (mgenov) [inline src/test/java/com/clouway/bank/core/SessionCleanerTest.java]: If you want to test the SessionCleaner you have to mock the session repository.

File: src/main/java/com/clouway/bank/BankBootstrap.java [hunk 2]
```diff
@@ -73,6 +69,14 @@ public static void main(String[] args) {
     Jetty jetty = new Jetty(httpPort, client);
     jetty.start();
 
+
+    //Cleaning Expired Sessions
+    SessionCleaner cleaner = new SessionCleaner(
+            new PersistentSessionRepository(Providers.of(client.getDatabase("bankApp"))),
+            300 //Every 5 minutes
+    );
+    cleaner.startAsync();
```
Comment (mgenov) [inline src/main/java/com/clouway/bank/BankBootstrap.java]: You can invoke: cleaner.startAsync().awaitRunning() to ensure that app is running.

Comment (mgenov) [inline src/main/java/com/clouway/bank/BankBootstrap.java]: The service need to be terminated gracefully when App is going to shutdown.

File: src/test/java/com/clouway/bank/core/SessionCleanerTest.java (hunk 1 above)
Comment (martinmilev) [inline src/test/java/com/clouway/bank/core/SessionCleanerTest.java]: fixed.

File: src/main/java/com/clouway/bank/BankBootstrap.java (hunk 2 above)
Comment (martinmilev) [inline src/main/java/com/clouway/bank/BankBootstrap.java]: fixed.

**Pull request merged**